Add button platform to Roborock (#29591)
home-assistant/home-assistant.github.io: source/_integrations/roborock.markdown

  - Switch
  - Time
  - Vacuum

>   - Button

Reviewer: Sorting

  - switch
  - time
  - vacuum

>   - button

Reviewer: Sorting

Volume - This allows you to control the volume of the robot's voice. For example, when it states "Starting cleaning". This allows you to set the volume to 0%, while the app limits it to 20%.

### Button

> There are currently four buttons that allow you to reset the various maintenance items on your vacuum. Pressing the button cannot be undone. For this reason,  they are disabled by default to make sure they are not pressed unintentionally.

Reviewer 2: ```suggestion
There are currently four buttons that allow you to reset the various maintenance items on your vacuum. Pressing the button cannot be undone. For this reason, the buttons are disabled by default to make sure they are not pressed unintentionally.
```

> There are currently four buttons that allow you to reset the various maintenance items on your vacuum. This cannot be undone after pressing - so they are disabled by default to make sure you know what you are doing before you click on them.

Reviewer 2: ```suggestion
There are currently four buttons that allow you to reset the various maintenance items on your vacuum. Pressing the button cannot be undone. For this reason,  they are disabled by default to make sure they are not pressed unintentionally.
```

> Reset sensor consumable - The sensors on your vacuum are intended to be cleaned after 30 hours of use

Reviewer 2: ```suggestion
Reset sensor consumable - The sensors on your vacuum are expected to be cleaned after 30 hours of use.
```

Reset side brush consumable - The side brush is expected to be replaced every 200 hours.

> Reset main brush consumable - the Main brush/ roller is expected ot be replaced every 300 hours.

Reviewer 2: ```suggestion
Reset main brush consumable - The main brush/ roller is expected to be replaced every 300 hours.
```

> Reset air filter - The Air filter is expected to be replaced every 150 hours.

Reviewer 2: ```suggestion
Reset air filter - The air filter is expected to be replaced every 150 hours.
```

> Reset air filter - the Air filter is expected to be replaced every 150 hours.

Reviewer 2: ```suggestion
Reset air filter - The Air filter is expected to be replaced every 150 hours.
```

> 

Reviewer 2: just to be sure I understand this correctly: pressing the reset button means the consumable was replaced / cleaned?

Author: Yep!
